staker/staker.go

```diff
@@ -36,6 +38,11 @@ const (
 	MakeNodesStrategy
 )
 
+var (
+	validatorWalletBalance      = metrics.NewRegisteredGaugeFloat64("arb/validator/wallet/balanceether", nil)
```

Comment:
This metric is now reflected in `arb/staker/balance` from https://github.com/OffchainLabs/nitro/pull/1463 . All the other metrics in this PR are still relevant though :) It'd also be good to transform that metric from an int64 to a float64 as you have it in this PR

Reply:
done.